activerecord/lib/active_record/connection_adapters/sqlite3/schema_dumper.rb

```diff
@@ -0,0 +1,13 @@
+module ActiveRecord
+  module ConnectionAdapters
+    module SQLite3
+      module ColumnDumper
+        private
+
+          def default_primary_key?(column)
+            schema_type(column) == :integer
+          end
```

Comment:
@kamipo Ah -- forgot that was added (this PR has been going on for awhile 😄 ). The parent class has it set as `bigint`, so we're locking SQLite at `integer`.